hydromt_sfincs/subgrid.py

```diff
@@ -18,13 +18,200 @@
 
 
 class SubgridTableRegular:
-    def __init__(self, version=0):
+    def __init__(self, version=1):
         # A regular subgrid table contains only for cells with msk>0
         self.version = version
 
-    def load(self, file_name, mask):
+    # new way of reading netcdf subgrid tables
+    def read(self, file_name, mask):
+        """Load subgrid table from netcdf file."""
+
+        self.version = 1
+
+        # Read data from netcdf file with xarray
+        self.ds = xr.open_dataset(file_name)
```

Comment:
do we need to store ds to self, i.e. do we use it outside this method? 
I would keep it as a local variable and move the close the the end of the method.

Reply:
Agree, no need to store it indeed, so I changed it. Before, I was hoping that we would have something like SubgridTableRegular.data, where this data whas the xr.DataArray that could be visualized/used righ-away.

However, since SFINCS expects flattened arrays with only values for the active cells, and also combines the u and v points into the same arrays, the NetCDF file is still not as usefull as hoped. I added the reading-in of the subgrid tables yesterday, so now it works in the same way as the binary files... Think good for now, but would be nice when we could store the subgrid in a NetCDF that can be visualized immediately.